README.md

```diff
@@ -352,6 +373,8 @@ rsync -azv <b>--exclude='kernels'</b> --delete --partial isisdist.astrogeology.u
 
 ### Mission Specific Data Downloads
 
+For versions of ISIS prior to ISIS 4.1.0, please use `isis3data` instead of `isisdata` in all the below rsync commands.
```

Comment:
Should we also mention cd is into ISIS3DATA instead of ISISDATA?

Reply:
This was a really good point that I missed. Thank you!